README.md

```diff
@@ -284,20 +284,15 @@ To get help, follow the instructions in the [Troubleshooting document](https://g
 
 ## Configuring a Proxy
 
-### HTTP Clients
+Google Cloud client libraries use HTTPS and gRPC in underlying communication
+with the services.
+In both protocols, you can configure a proxy using `https.proxyHost`
+and (optional) `https.proxyPort` properties.
 
-For HTTP clients, a proxy can be configured by using `http.proxyHost`, `https.proxyHost`, and
-related system properties as documented by
```

Comment:
Memo: "related system properties" in [the document](https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html) for HTTPS was just "https.proxyPort".